examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -203,7 +208,7 @@ public static void performance_calibration_pricing() {
       for (CurveNode node : nodes) {
         if (!(node instanceof IborFixingDepositCurveNode)) {
           // IborFixingDeposit is not a real trade, so there is no appropriate comparison
-          trades.add(node.trade(VALUATION_DATE, ObservableValues.ofIdMap(quotes)));
+          trades.add(node.trade(VALUATION_DATE, MarketData.builder().addValues(quotesByKey).build()));
```

Comment:
As things stand `addValuesById` could only accept `ObservableId`. `ObservableId` can be converted to an `ObservableKey` but `MarketDataId` doesn't have a method (yet) to convert it to a `MarketDataKey`. It would be possible to add one but this is the only case I've seen so far where it would be useful. It's something to think about.